js/task17.js

```diff
@@ -0,0 +1,126 @@
+const ul = document.getElementById('lists');
+
+let currentImage = 0;
+const imagesArr = [];
+
+function showLoader() {
+	const li = document.createElement('li');
+	const loader = document.createElement('img');
+	li.classList.add('loader');
+	loader.classList.add('loader_image');
+	loader.src = './img/loading-circle.gif';
+	ul.appendChild(li).appendChild(loader);
+}
+function removeLoader() {
+	const loaderImage = document.querySelector('.loader');
+	ul.removeChild(loaderImage);
+}
+
+function fetchData() {
+	return new Promise((resolve, reject) => {
+		setTimeout(() => {
+			resolve(fetch('./js/json/task17.json'));
+		}, 3000);
+	});
+}
+
+async function getData() {
+	showLoader();
+
+	// let response, res, data;

```

Comment:
nits:
I thought it would be better to delete the comment.

Reply:
Sure, I'll delete it.